commands/fun/vaportype.js

```diff
@@ -0,0 +1,24 @@
+const Command = require(`${process.cwd()}/base/Command.js`);
+class Vaportype extends Command {
+  constructor(client) {
+    super(client, {
+      name: "vaportype",
+      description: "W O A H  D U D E  S P A C E D  O U T",
+      usage: "vaportype <text>",
+      category: "fun",
+      aliases: ["vapor", "vpt"]
+    });
+  }
+
+  async run(message, args, level) { // eslint-disable-line no-unused-vars
```

Comment:
this can be `async run(message, args) {`

Reply:
Wasn't sure if that would work with how we execute commands, went ahead and did it anyway.